<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_out</class>
 <widget class="QDialog" name="log_out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QLabel {
background-color: white
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>1391</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="log_out_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>711</width>
     <height>721</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
background-color: white;
}

.QPushButton:hover {
background-color: black;
color: white
}</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>-20</y>
     <width>711</width>
     <height>751</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
background-color: white;
}

.QPushButton:hover {
background-color: black;
color: white
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
